### PR TITLE
Add EZMultiPli to product database. 

### PR DESCRIPTION
**Pull request by mkowalchuk**
Based on user manual at http://www.expresscontrols.com/pdf/EZMultiPliOwnerManual.pdf and the Z-Wave db entry at www.pepper1.net/zwavedb/device/592.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/homeseer/ezmultipli.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			<Label lang="en">TempAdj</Label>
+			<Help lang="en"><![CDATA[
+TempAdj is a twos-complement number that is used to adjust the temperature reading to make it more accurate. The value is in tenths of degree Fahrenheit. The temperature reading can be adjusted by +12.7F to -12.8F. A value of 1 will adjust the temperature reading by +0.1F. A value of -1 will adjust the temperature by -0.1F. A value of 123 will adjust the temperature by +12.3F. 
+			]]></Help>
+		</Parameter>
+	</Configuration>
+
+	<Associations>
+		<Group>
+			<Index>1</Index>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/homeseer/ezmultipli.xml]: This should have the < SetToController > element set to true

File: bundles/binding/org.openhab.binding.zwave/database/homeseer/ezmultipli.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+			<Maximum>1</Maximum>
+			<Label lang="en">LIFELINE</Label>
+			<Help lang="en"><![CDATA[
+This group is normally automatically assigned to the Z-Wave system controller when EZMultiPli is added to the ZWave network. All Notification command class reports (motion detection) are sent to the node assigned to this group. Advanced lighting control should be handled by a home control application based on these reports.			
+			]]></Help>
+		</Group>
+		<Group>
+			<Index>2</Index>
+			<Maximum>4</Maximum>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/homeseer/ezmultipli.xml]: If the lifeline doesn't also get reports when the light comes on, then this shoudl also have the SetToController true

**Comment by cdjackson** [comment]
Just a couple of comments before I merge.

Please can you also look at adding this to the device database so that it will also be available in OH2 -;
http://www.cd-jackson.com/index.php/zwave/zwave-device-database

**Comment by mkowalchuk** [comment]
Added SetToController to the association groups.

I'd be glad to add it to the OH2 device database. Do I need any special permissions to do so? My view of the Device List page doesn't have the Add button that http://www.cd-jackson.com/images/zwave/zwaveDatabase-add.png shows.

**Comment by cdjackson** [comment]
Thanks - I’ve added you to the editors group, so you should now have the add button :)

Any problems, just shout…

Thanks
Chris

**Pull request merged**